install/common/pagespeed.conf.template

```diff
@@ -321,15 +321,19 @@
     # and change server state, such as statistics, caches, and
     # messages.  This might be appropriate in an experimental setup.
     <Location /pagespeed_admin>
-        Order allow,deny
-        Allow from localhost
-        Allow from 127.0.0.1
+        Require all granted
+        Require local
```

Comment:
Are these two 'Require' lines equivalent to the above three that you removed?

Reply:
can you put these under ifdef so this still works on 2.2?  Or will it work on 2.2 as is?

Or are we deciding to remove support for 2.2?

Reply:
Apache 2.2 still looks pretty relevant, i would vote for continued support?:
https://trends.builtwith.com/Web-Server/Apache-2.2

Reply:
Hello,
@oschaaf @jmarantz 
>Are these two 'Require' lines equivalent to the above three that you removed?
yes, they are.

Reply:
>can you put these under ifdef so this still works on 2.2? Or will it work on 2.2 as is?

I will put ifdef, as it can't work with apache 2.2.

Apache 2.2 directives may be kept in 2.4, but may have "weird" unexpected behavior, so ifdef is the best solution.

Reply:
```
# Attempt to load mod_version if it wasn't loaded or compiled in (eg on Debian)
<IfModule !mod_version.c>
LoadModule version_module /usr/lib/httpd/modules/mod_version.so
</IfModule>
<Location /pagespeed_admin>
        <IfVersion <2.4>
            Order allow,deny
            Allow from localhost
            Allow from 127.0.0.1
            <IfVersion>
        <IfVersion >= 2.4>
            Require all granted
            Require local
            #Require ip, host or forward-dns  - see https://httpd.apache.org/docs/current/fr/mod/mod_authz_host.html
            </IfVersion>
        SetHandler pagespeed_admin
    </Location>
    <Location /pagespeed_global_admin>
        <IfVersion <2.4>
            Order allow,deny
            Allow from localhost
            Allow from 127.0.0.1
            <IfVersion>
        <IfVersion >= 2.4>
            Require all granted
            Require local
	     #Require ip, host or forward-dns
            </IfVersion>
          SetHandler pagespeed_global_admin
    </Location>
```